config/services/dataplane_v1beta1_openstackdataplaneservice_nova.yaml

```diff
@@ -18,3 +18,6 @@ spec:
     - ctlplane
     issuer: osp-rootca-issuer-internal
   caCerts: combined-ca-bundle
+  containerImageFields:
+  - NovaComputeImage
+  - EdpmIscsidImage
```

Comment:
I'm not sure this is the complete list. The osp.edpm.nova_storge playbook uses not just the iscsid but also the 
openstack-multipathd image as well.

Reply:
I don't see that image in config/default/manager_default_images.yaml either. Maybe we need to add it?

Reply:
Added here: https://github.com/openstack-k8s-operators/dataplane-operator/pull/811
and here for openstack-operator: https://github.com/openstack-k8s-operators/openstack-operator/pull/741
once the openstack-operator PR merges, I can update this one.

Reply:
shouldn't it be appended here?